source/includes/fundamentals/code-snippets/srv.go

```diff
@@ -29,10 +30,10 @@ func main() {
 		}
 	}()
 
-	// Ping the primary
+	// Send a ping to confirm successful connection
 	if err := client.Ping(context.TODO(), readpref.Primary()); err != nil {
 		panic(err)
 	}
 
-	fmt.Println("Successfully connected and pinged.")
+	fmt.Println("Pinged the primary node of the cluster. You are connected to MongoDB!")
```

Comment:
Suggestion:

I wonder if it might be confusing to mention that "you are connected" and then immediately disconnect after that. What do you think about "You successfully connected to MongoDB!" to replace the second sentence?